programs/zstd.1.md

```diff
@@ -128,6 +128,12 @@ the last one takes effect.
     selection, namely that windowSize > srcSize.
 
     Note: cannot use both this and -D together
+    Note: consider using --patch-from with `--long` mode on large files
```

Comment:
Since `--long` mode is automatically activated when considered useful, I guess it's not that important to suggest to the user that it should do it manually.
Maybe it's still useful to suggest that the manual option is available.
In which case, I would recommend to swap the order of notes 2 & 3, or combine them.